doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,331 @@
+.. _migration_cs3_to_2_6_99_cs2_env:
+
+Update your development environment for |NCS| v2.6.99_cs2 (for v2.4.99-cs3 users)
+#################################################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This document describes how to update your development environment from |NCS| v2.4.99-cs3 to |NCS| v2.6.99_cs2.
+
+The main development environment changes introduced by 2.6.99-cs2 for the nRF54H20 DK are the following:
+
+* The |NCS| toolchain has been updated.
+  A bootstrap script specific to the nRF54H20 DK is now available to install and update the toolchain.
+* nRF Util has now replaced nRF Command Line Tools.
+  The bootstrap script will now install and update nRF Util.
+* SDFW and SCFW are now provided as precompiled binaries.
+  The Secure Domain Firmware (SDFW) and System Controller Firmware (SCFW) are no longer built from the source during the application build process, but they are installed as binaries from the provided firmware bundle and programmed when flashing the application to the device.
+  See the details in the `Installing the nRF54H20 firmware bundle`_ section below.
+* The nRF54H20 SoC lifecycle state must now be set to Root of Trust (RoT).
+  See the details in the `Set the nRF54H20 to RoT`_ secton.
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer has one of the supported operating systems.
+
+Hardware
+========
+
+* nRF54H20 DK version PCA10175 v0.7.0 or v0.8.0 (ES3).
+  These are the only versions of the nRF54H20 DK compatible with |NCS| 2.6.99-cs2.
+  Check the version number on your DK's sticker to verify its compatibility with |NCS| version 2.6.99-cs2.
+* USB-C cable
+
+Software
+========
+
+On your computer, one of the following operating systems:
+
+* Microsoft Windows
+* Ubuntu Linux
+* macOS
+
+|supported OS|
+
+You also need the following:
+
+* `Git`_ or `Git for Windows`_ (on Linux and Mac, or Windows, respectively)
+* `curl`_
+
+Installing the new toolchain
+****************************
+
+You can install the new toolchain for the |NCS| v2.6.99-cs2 by running an installation script.
+Ensure that you additionally have Git and curl installed.
+
+.. tabs::
+
+   .. tab:: Windows
+
+      Follow these steps:
+
+      1. Open Git Bash.
+      #. Download and run the :file:`bootstrap-toolchain.sh` installation script file using the following command:
+
+         .. parsed-literal::
+            :class: highlight
+
+            curl --proto '=https' --tlsv1.2 -sSf https://files.nordicsemi.com/artifactory/swtools-staging/external/scripts/bootstrap-toolchain.sh | NCS_TOOLCHAIN_VERSION=v2.6.99-cs2 sh
```

Comment:
Update to:
`curl --proto '=https' --tlsv1.2 -sSf https://files.nordicsemi.com/artifactory/swtools/external/scripts/bootstrap-toolchain.sh | NCS_TOOLCHAIN_VERSION=v2.6.99-cs2 sh`